src/main/java/igym/services/WorkoutService.java

```diff
@@ -121,6 +125,24 @@ private void deleteExerciesList(Workout workout) {
         }
     }
 
+    /**
+     * Soft deletes an exercise by marking its status as {@code Status.inactive}.
+     *
+     * @param exerciseId the UUID of the exercise to soft delete
+     * @throws WorkoutNotFoundException if the workout or exercise with the provided
+     *                                  IDs do not exist or are already inactive
+     */
+    @Transactional
+    public void deleteExercise(UUID exerciseId) {
+        logger.info("Attempting to inactivate exercise with id: {}", exerciseId);
+    
+        Exercise exercise = exerciseRepository.findByIdAndStatus(exerciseId, Status.active)
+                .orElseThrow(() -> new ExerciseNotFoundException("Exercise with id " + exerciseId + " not found"));
+    
+        exercise.setStatus(Status.inactive);
+        logger.info("Exercise with id {} has been inactivated", exerciseId);
```

Comment:
```suggestion
        logger.info("Exercise with id {} inactivated", exerciseId);
```

to follow the same pattern